source/objects/waveClass.m

```diff
@@ -706,15 +703,15 @@ function waveElevUser(obj,rampTime,dt,maxIt,data,time)
             obj.waveAmpTime(:,1) = time;
             obj.waveAmpTime(:,2) = rampFunction.*interp1(data_t,data_x,time);
             
-            if any(~isempty(obj.markerLoc))
+            if any(~isempty(obj.marker.location))
                 warning('Cannot use wave gauges or visualization markers with eta import. Gauges and markers removed.');
-                obj.markerLoc = [];
+                obj.marker.location = [];
             end
         end
         
         function printWaveSpectrumType(obj)
             % Lists the wave spectrum type
-            % Used by listInfo
+            % Used by  :meth:`waveClass.listInfo`.

```

Comment:
should this be ``:math:`waveCLass.listInfo`.``?

Reply:
`:meth:` refers to method